packages/message-relayer/src/service.ts

```diff
@@ -281,6 +312,22 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
 
       this.state.eventCache = this.state.eventCache.concat(events)
       startingBlock += this.options.getLogsInterval
+
+      // tslint:disable-next-line
+      const event = this.state.eventCache.find((event) => {
+        return (
```

Comment:
This line is pretty verbose. Also should we consider using a `Map` for the cache?

Reply:
We could. For now I just refactored this logic a bit to make it a little more clear.